aws/modules/database/parameter_group.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_db_parameter_group" "db" {
+  name   = "${var.id}"
+  family = "postgres9.5"
+
+  parameter {
+    name = "log_hostname"
+    value = "0"
```

Comment:
Can we do off instead of 0 to make it more clear?

https://gpdb.docs.pivotal.io/4370/guc_config-log_hostname.html